bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/MathUtils.java

```diff
@@ -63,4 +63,28 @@ public static long remainder(long numerator, long denominator) {
             }
         }
     }
+
+    public static long addExact(long num1, long num2) {
+        try {
+            return Math.addExact(num1, num2);
+        } catch (ArithmeticException e) {
+            throw ErrorCreator.createError(BallerinaErrorReasons.NUMBER_OVERFLOW, INT_RANGE_OVERFLOW_ERROR);
```

Comment:
what I normally would do to avoid duplication is. Write a function
```
public BError getNumberOverFlowError() {
   return ErrorCreator.createError(BallerinaErrorReasons.NUMBER_OVERFLOW, INT_RANGE_OVERFLOW_ERROR);
}
```
and throw the error in the catch statement
```
throw getNumberOverFlowError();
```

Reply:
I will look into these.

Reply:
When I try to do the above _jballerina-debugger-integration-test_ fails.

Reply:
ErrorUtils class we already use create internal errors. ex.createNumericConversionError. We can add the method there and call. 

throw ErrorUtils.createNumberOverFlowError(...)

Reply:
I will look into this.